src/main/java/module-info.java

```diff
@@ -59,7 +59,7 @@
     requires jbibtex;
     requires citeproc.java;
     requires antlr.runtime;
-    requires commons.lang3;
```

Comment:
Why was `commons.lang3´ removed?

Reply:
Because I got an error message saying that the module does not exist.